src/test/java/raisetech/student/management/repository/StudentRepositoryTest.java

```diff
@@ -194,19 +194,19 @@ private static StudentCourse createStudentCourse(Student student){
   @Test
   void 受講生コース情報のコース名の更新が行えること(){
     int studentId = 5;
-    List<StudentCourse> studentCourses = sut.searchStudentCourse(String.valueOf(studentId));
+    List<StudentCourse> studentCourses = sut.searchStudentCourse(java.lang.String.valueOf(studentId));
     studentCourses.get(0).setCourseName("AWSコース2");
 
     sut.updateStudentCourse(studentCourses.get(0));
 
-    List<StudentCourse> actual = sut.searchStudentCourse(String.valueOf(studentId));
+    List<StudentCourse> actual = sut.searchStudentCourse(java.lang.String.valueOf(studentId));
     assertEquals("AWSコース2", actual.get(0).getCourseName());
 
   }
 
   @Test
   void コース申込状況の更新が行えること(){
-    String courseId = "1";
+    java.lang.String courseId = "1";
```

Comment:
```suggestion
    String courseId = "1";
```
でよいです。
もしjava.langを書かないとコンパイルエラーになるということであれば不自然なので原因究明したほうがいいですね。